docker/types/services.py

```diff
@@ -334,9 +334,12 @@ class UpdateConfig(dict):
         max_failure_ratio (float): The fraction of tasks that may fail during
           an update before the failure action is invoked, specified as a
           floating point number between 0 and 1. Default: 0
+        order (string): Specifies the order of operations when rolling out an
+          updated task. Either ``start_first`` or ``stop_first`` are accepted.
+          Default: ``stop_first``
     """
     def __init__(self, parallelism=0, delay=None, failure_action='continue',
-                 monitor=None, max_failure_ratio=None):
+                 monitor=None, max_failure_ratio=None, order='stop-first'):
```

Comment:
This should be `None` by default, otherwise any client using API version < 1.29 will fail unless the user explicitly unsets it.